setup.py

```diff
@@ -1,13 +1,48 @@
-from setuptools import setup, find_packages
+# -*- coding: utf-8 -*-
```

Comment:
Missing SPDX tag

Reply:
setup.py is a default file, like Makefile, manifest.in and so on, do that files need the SPDX as well?

Reply:
Is this file generated?

Reply:
As much as Makefile, manifest.in and so on...

Reply:
The only files which clearly don't need a license header are generated files. All other files should contain a license header. It is not a big issue, but I was just a minute to late to realize the issue. I had written the comment but didn't hit the button until you merged the branch. So ...

Reply:
Link the law, don't see the need in that kind of files, you can autogenerate all of that... So no SPDX needed...

Reply:
BUT: What about Markdown files? Do we have to add a SPDX commend? @iluminat23

Reply:
Hmmm, this is a good question. A GPL or MIT would not do any good with these kind of files. And I have no idea what license our documentation should have.